app/controllers/api/v1/tasks_controller.rb

```diff
@@ -0,0 +1,58 @@
+class Api::V1::TasksController < Api::V1::ApplicationController
+  def index
+    q_params = params[:q] || { s: 'id asc' }
+
+    tasks = Task.all
+      .ransack(q_params)
+      .result
+      .page(params[:page])
+      .per(params[:per_page])
+
+    json = {
+      items: serialize(tasks, TaskSerializer),
```

Comment:
ActiveModelSerializer can serialize you collections by himself you just need to pass serializer to `serializer` or `each_serializer` option in the render method